server/models/postgis/task.py

```diff
@@ -134,8 +134,13 @@ def as_dto(self):
         issue_dto.count = self.count
         return issue_dto
 
-    def __repr__(self):
```

Comment:
Why did you remove this `__repr__` method?

Reply:
With an approach I was trying at first, I needed mapping issues to be returned from the database as objects so I could access their members, including mapping_issue_category_id. (I was using the two methods I wrote here, so issues were only returned as the string "issue: count".) There were no calls to any methods that used this __repr__ method, so I removed it.

Now I've implemented the functionality a different way, so it makes no difference for the issues export functionality. I've realized all of the methods here (__repr__, get_all_issues(), get_issue_row_count()) are now unused, so I plan to remove the two I wrote. I can add the __repr__ method back in if you'd like, but it's still unused.

Reply:
This `__repr__` method is in v3.4.9 of the TM. I would leave it in.